rules-tests/DeadCode/Rector/ClassMethod/RemoveUselessParamTagRector/Fixture/remove_null.php.inc

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Rector\Tests\DeadCode\Rector\ClassMethod\RemoveUselessParamTagRector\Fixture;
+
+class RemoveNull
+{
+    /**
+     * @param null $a
+     */
+    public function foo($a = null)
+    {
+
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Tests\DeadCode\Rector\ClassMethod\RemoveUselessParamTagRector\Fixture;
+
+class RemoveNull
+{
+    public function foo($a = null)
```

Comment:
same here, it cause no param type specified phpstan notice, on big project, it cause a lot of notice